gen_template.rb

```diff
@@ -82,6 +82,29 @@ def gen_template(
           ['-m', '{{ user `memory` }}'],
         ],
       ),
+      builder(
+        type: 'hyperv-iso',
+        generation: 1,
+        # generation: 2, # gen 2 fails uefi boot, missing configuration in builders/hyperv-iso.nix
+        iso_url: iso_url,
+        iso_checksum: iso_sha256,
+        boot_wait: "60s",
+        boot_command: [
+          "echo http://{{ .HTTPIP }}:{{ .HTTPPort}} > .packer_http<enter>",
+          "mkdir -m 0700 .ssh<enter>",
+          "curl $(cat .packer_http)/install_rsa.pub > .ssh/authorized_keys<enter>",
+          "sudo nix-env -f '<nixpkgs>' -iA linuxPackages.hyperv-daemons<enter><wait10>",
```

Comment:
```suggestion
          "sudo nix-env -iA linuxPackages.hyperv-daemons<enter><wait10>",
```
I think this bit is superfluous

Reply:
Ah yes, I believe you're right, will cleanup and test.